code/modules/mob/living/simple_animal/hostile/human/surgeon.dm

```diff
@@ -0,0 +1,71 @@
+/mob/living/simple_animal/hostile/humanoid/surgeon/
+	name = "\improper Surgeon"
+	desc = "Don't interfere with his work!"
+	icon = 'icons/mob/hostile_humanoid.dmi'
+	icon_state = "surgeon"
+	icon_living = "surgeon"
+	maxHealth = 100
+	health = 100
+
+	harm_intent_damage = 10
+	melee_damage_lower = 8
+	melee_damage_upper = 13
+	attacktext = "saws"
+	attack_sound = 'sound/weapons/circsawhit.ogg'
+
+	ranged = 0
+	gender = MALE
+
+	//corpse = /obj/effect/landmark/corpse/surgeon
+	items_to_drop = list(/obj/item/weapon/circular_saw/plasmasaw)
+
+/mob/living/simple_animal/hostile/humanoid/surgeon/boss
+	name = "Doctor Placeholder" //I can't think of a good name
+	desc = "A deranged surgeon lost deep in space."
+	icon_dead = null //this isn't even his final form
+	maxHealth = 80
+	health = 80
+
+	harm_intent_damage = 15
+	melee_damage_lower = 13
+	melee_damage_upper = 20
+
+	corpse = null //he isn't dead yet
+
+	faction = "necro" //don't want him to get killed by skeletons before his transformation
+
+/mob/living/simple_animal/hostile/humanoid/surgeon/boss/Die()
+	visible_message("<span class=danger><B>Before he can die, the mad surgeon takes a drink of a foul-smelling concoction and begins to mutate! </span></B>")
+	say("[pick("YOU CAN'T KILL ME THAT EASILY!", "I WONT LET YOU STOP ME!", "GET OUT OF MY FACILITY!", "I MUST CONTINUE MY RESEARCH!", "I'M GONNA WRECK IT!", "I'VE GOT A BONE TO PICK WITH YOU!")]")
+	var/obj/effect/effect/smoke/S = new /obj/effect/effect/smoke(get_turf(src))
```

Comment:
this will only make a single bit of smoke, are you sure you don't want it to spread?

If you do, look at /datum/effect/effect/system/smoke_spread